### ISSUE TITLE
[Suggestion] Broadcast Connection

### ISSUE THREAD

**Issue by Deliphin** [CTurt/3DSController#59]
Instead of depending on a single IP address, is it possible to make the 3DS application send a broadcast to the whole LAN, and when the server receives it, it connects to the 3DS app?

This would have issues if you have more than one computer running a 3DSController server, though. Maybe when the 3DS application receives multiple different connections, it could show a list of choices using the hostnames of the computers?

**Comment by ElectroJo** [comment]
You can already do this by using your networks broadcast address in the 3ds side .ini file. For example, if your on the network 192.168.0.1 with a broadcast of 255.255.255.0, you can use the ip address 192.168.0.255 in the ini file and it will broadcast to every device on the network.

However your issue of multiple computers still stands, plut it will  to broadcast to everything on the network the whole time.

**Comment by Deliphin** [comment]
I didn't think of that, that sounds really useful. However it's not the same as what I'm asking for, since as you said, it broadcasts everywhere all the time, you can't have multiple 3DS controllers, as well as if you go to another network that for some reason has a different broadcast address (maybe it's on a subnet or something), then it won't work.

**Comment by ElectroJo** [comment]
Yeah I understand, I was just providing a temporary solution to the first paragraph since this project hasn't been updated in a while (Sadly). This suggestion is likely better put in one of the many forks as several have been updated as soon as a few months ago (while this was last updated 2 years ago).
